lib/galaxy/auth/__init__.py

```diff
@@ -15,6 +15,7 @@ class AuthManager(object):
 
     def __init__(self, app):
         self.__app = app
+        self.redact_username_in_logs = app.config.redact_username_in_logs
```

Comment:
this is shared with the Tool Shed which does not have this in its config object

Reply:
ugh, forgot about reports + TS. will implement these this afternoon :(

Reply:
I've added the config options but struggling to launch a working TS.

![auswahl_138](https://user-images.githubusercontent.com/458683/39991102-f2662808-575d-11e8-8d22-c35974e28342.png)

Using only the stock shed config.

Reply:
Nope. ignore me. I was being dumb.